build.gradle

```diff
@@ -91,6 +89,7 @@ dependencies {
     swaggerCodegen group: 'io.swagger.codegen.v3', name: 'swagger-codegen-cli'
     runtimeOnly group: 'org.webjars.npm', name: 'swagger-ui-dist', version: '3.35.2'
     implementation group: 'io.swagger.core.v3', name: 'swagger-annotations'
+    implementation group: 'io.swagger', name: 'swagger-annotations', version: '1.6.2'
```

Comment:
I think this version is managed by the Spring dependency plugin block above, so you shouldn't need to specify the version here or in `terra-resource-janitor-client/build.gradle`

Reply:
Good catch, thanks!